query/src/main/java/org/infinispan/query/dsl/embedded/impl/JPQLFilterAndConverter.java

```diff
@@ -25,7 +25,7 @@
  * @author anistor@redhat.com
  * @since 7.0
  */
-public final class FilterAndConverter<K, V> extends AbstractKeyValueFilterConverter<K, V, ObjectFilter.FilterResult> {
+public final class JPQLFilterAndConverter<K, V> extends AbstractKeyValueFilterConverter<K, V, ObjectFilter.FilterResult> {
```

Comment:
In many places you have s/JPQL/JPA/ but here you're using JQPL. For consistency stick to one :)